relays/utils/src/metrics.rs

```diff
@@ -31,62 +32,32 @@ pub struct MetricsParams {
 }
 
 /// Metrics API.
-pub trait Metrics {
+pub trait Metrics: Clone + Send + Sync + 'static {
```

Comment:
Why does this need to be `'static` now?

Reply:
It is now used in [`async_std::task::spawn()`](https://docs.rs/async-std/1.9.0/async_std/task/fn.spawn.html), which required future to be `'static`